src/services/ProfileService.ts

```diff
@@ -48,22 +48,26 @@ const useSetFavoriteCommander = () => {
     const endpoint = "https://chatterfang.onrender.com/profiles";
 
     return useCallback(
-        (commanderId: string) => {
+        (commanderId: string, moxfieldId?: string) => {
+            const newProfile = moxfieldId
```

Comment:
this is probably ok for now, but won't scale as we add more properties to the profile. something to think about for the future